lib/utils/constants.ts

```diff
@@ -40,3 +40,6 @@ export type TransistorPorts =
   | "base"
 
 export type { BaseSymbolName }
+
+export const SCHEMATIC_COMPONENT_OUTLINE_COLOR = "rgba(132, 0, 0)"
+export const SCHEMATIC_COMPONENT_OUTLINE_STROKE_WIDTH = 0.12
```

Comment:
i would leave stroke width default to circuit-to-svg (again, for scale invariant rendering)